pom.xml

```diff
@@ -172,6 +172,11 @@
             </plugins>
         </pluginManagement>
         <plugins>
+          <plugin>
+            <groupId>org.ensime.maven.plugins</groupId>
+            <artifactId>ensime-maven</artifactId>
```

Comment:
are they necessary?

Reply:
Maybe this should move to maven profile.